go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/prometheus-operator/prometheus-operator
 
-go 1.22
```

Comment:
could you please revert it?

Reply:
Sure. I was getting errors with resolving this in my local machine, hence the change. I will revert it.